pyquil/tests/test_job_results.py

```diff
@@ -17,3 +21,16 @@ def test_octet_bits():
     assert [0, 1, 0, 0, 0, 0, 0, 0] == _octet_bits(0b10)
     assert [1, 0, 1, 0, 0, 0, 0, 0] == _octet_bits(0b101)
     assert [1, 1, 1, 1, 1, 1, 1, 1] == _octet_bits(0b11111111)
+
+
+def test_get_is_called():
+    # check that JobResult.is_done() calls .get() on QPU prior to
+    # returning result
+    qpu_connect = QPUConnection('test')
+    job_result = JobResult(qpu_connect, False)
+    try:
```

Comment:
the get() method has a call to qpu.get_job(), but I didn't know if there was a way to setup a qpu object without forcing a Forest API connection, so the get() will throw an exception that mock can then identify. is there a cleaner workaround (so the qpu.get_job() call resolves)?